flink-cdc-connect/flink-cdc-source-connectors/flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/source/fetch/PostgresSourceFetchTaskContext.java

```diff
@@ -302,8 +301,11 @@ public Offset getStreamOffset(SourceRecord sourceRecord) {
     }
 
     @Override
-    public void close() {
+    public void close() throws Exception {
         jdbcConnection.close();
+        if (replicationConnection != null) {
```

Comment:
check null for `jdbcConnection` as well ?

Reply:
done it